assets/js/components/settings/SettingsModule.js

```diff
@@ -89,31 +92,39 @@ class SettingsModule extends Component {
 		global.removeEventListener( 'keyup', this.handleCloseModal );
 	}
 
+	componentDidUpdate( prevProps ) {
+		const { slug } = this.props.module;
+		const { moduleSlug: previousSlug } = prevProps.match.params;
+		const { moduleSlug: currentSlug } = this.props.match.params;
+		const changingModule = ( previousSlug === slug ) || ( currentSlug === slug );
+		const openModuleChanged = currentSlug !== previousSlug;
```

Comment:
I follow what's happening here but it might be a bit opaque to someone new to the project or the component. Could you provide a bit more documentation in a comment here for why this flow occurs and what we're resetting when this happens? 🙂